src/index.js

```diff
@@ -404,6 +413,8 @@ export {
   SaveAs,
   enableLogger,
   disableLogger,
+  calculateSUV,
+  convertToVector3,
```

Comment:
These are `lib` exports:

- [convertToVector3](https://github.com/cornerstonejs/cornerstoneTools/blob/master/src/lib.js#L154)
- [calculateSUV](https://github.com/cornerstonejs/cornerstoneTools/blob/master/src/lib.js#L135)

You can import them like this:

```js
import { importInternal } from 'cornerstone-tools';

const convertToVector3 = importInternal('util/convertToVector3');
```

Reply:
I wasn't aware of this function! I will remove this from the pull request.